### PR TITLE
GraphQL

### PR DESCRIPTION
**Pull request by martindale**
GraphQL support at `/services/graphql` and some general cleanup

## Notable Changes
- Migrates compiled applications to `production` mode — much smaller output!

### Ready for Review (RFR)
- [ ] restore `INVENTORY_REQUEST` message in Remote

### CODE REVIEW

File: types/server.graphql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+type FabricHTTPServer {
+  id: ID!
+  alias: String
+  host: String
+  port: Int
+  pubkey: String
+  services: [Service]
```
Comment (chrisinajar) [inline types/server.graphql:7]: are these all really optional?

File: types/server.js [hunk 2]
```diff
@@ -709,23 +739,42 @@ class FabricHTTPServer extends Service {
...
+        name: 'Query',
+        fields: {
+          hello: {
+            type: GraphQLString,
+            resolve: () => 'world',
+          }
+        }
+      })
+    });
+
```
Comment (chrisinajar) [inline types/server.js:774]: hmm....

File: types/spa.js [hunk 3]
```diff
@@ -231,11 +251,11 @@ class SPA extends App {
   <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
   <title>${this.title}</title>
   <!-- <link rel="manifest" href="/manifest.json"> -->
-  <link rel="stylesheet" type="text/css" href="/styles/screen.css" />
   <link rel="stylesheet" type="text/css" href="/styles/semantic.css" />
-  <script type="text/javascript" src="/scripts/jquery-3.4.1.js"></script>
-  <script type="text/javascript" src="/scripts/semantic.js"></script>
-  <script src="bundles/browser.js"></script>
+  <link rel="stylesheet" type="text/css" href="/styles/screen.css" />
+  <script src="/scripts/jquery-3.4.1.js"></script>
```
Comment (chrisinajar) [inline types/spa.js:256]: O.O

Comment (martindale) [reply types/spa.js:256]: This might go away with `fomantic-ui-react` — otherwise see Semantic-Org/Semantic-UI#1175

**Pull request merged**